target/linux/ramips/image/mt7620.mk

```diff
@@ -629,6 +629,21 @@ define Device/netgear_ex3700
 endef
 TARGET_DEVICES += netgear_ex3700
 
+define Device/netgear_ex6100-v1
+  SOC := mt7620a
+  NETGEAR_BOARD_ID := U12H248T00_NETGEAR
+  BLOCKSIZE := 4k
+  IMAGE_SIZE := 7744k
+  IMAGES += factory.chk
+  IMAGE/factory.chk := $$(sysupgrade_bin) | check-size $$$$(IMAGE_SIZE) | \
```

Comment:
check-size can be used without argument now. This will allow you to remove the line wrap as well.